.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Motivation
+![gif](gif.gif)
```

Comment:
You should remove this though - i think it'll render this into a broken image. To add a gif you need to literally drag and drop file into PR.